src/content/self-hosted/install/releases.mdx

```diff
@@ -5,6 +5,22 @@ sidebar_label: Release Notes
 id: releases
 ---
 
+## 1.3.0
+
+### New Feature
+
+- Git Catalog to provide predefined development environments to every developer
+- Add Git Catalog tab to Okteto UI deploy
```

Comment:
These two are the same right? I think it's better to put it on a single item. Also @omnomagonz @pchico83   could you help with a more user-friendly description? This is a powerful feature, I think we should mention a bit more why you should use this and how it benefits users.

Reply:
My attempt at more user-friendly wording:

> Git Catalog that enables admins to pre-define a list of development environments that developers can use to deploy an environment without configuring it themselves

Something like that? I think we can (and should) expand further in the docs we create for this

Reply:
> These two are the same right? …

Yes.

> … I think we can (and should) expand further in the docs we create for this

+1 - I guess as a general rule: release notes for the _what_, link to docs for the _how_ (to use it)? :)

Reply:
> release notes for the what, link to docs for the how (to use it)?

For now, yep! In the future, I think it'd be great to do something like what we did at GitLab where the "release notes" were basically a blog post. For each major feature we explained:

* What is it?
* Why did we build it?
* Which customers (pricing tiers) get access to it
* A screenshot showing the feature
* A link to the epic or issue for it (if this becomes public later)

The minor features (and non-customer facing ones) were a bulleted list at the bottom.

All that to say: I'd love to eventually see the "what" _and_ "why" in the release notes, but a more thorough documentation or explanation of the feature done elsewhere. In the future anyway :)

Reply:
@omnomagonz this is great, could you create a notion doc with these questions (maybe an entry inproduct on release notes guidance) so they don't get lost after we close this PR?

Reply:
@rberrelleza you bet! [Here you go](https://www.notion.so/okteto/Monthly-Release-Notes-2afbe7e3e3c04c48aa99075331c07665)